libs/net/dal/Services/FileReferenceService.cs

```diff
@@ -225,20 +218,24 @@ public async Task<bool> UploadToS3Async(string s3Key, Stream fileStream)
         }
     }
 
-    public async Task<IEnumerable<FileReference>> GetFiles(DateTime? updatedBefore = null, int limit = 100)
+    public async Task<IEnumerable<FileReference>> GetFiles(DateTime? createdBefore = null, int limit = 100, bool force = false)
     {
         try
         {
             IQueryable<FileReference> query = this.Context.FileReferences;
 
-            if (updatedBefore.HasValue)
+            if (createdBefore.HasValue)
             {
-                updatedBefore = updatedBefore.Value.ToUniversalTime();
-                query = query.Where(fr => fr.UpdatedOn < updatedBefore.Value);
+                createdBefore = createdBefore.Value.ToUniversalTime();
+                query = query.Where(fr => fr.CreatedOn < createdBefore.Value);
```

Comment:
Why are we searching for files created before a date?

Reply:
yes, we get a file before a date

Reply:
> Why are we searching for files created before a date?

We don't want to move the latest files to the S3 bucket, might be move files before 30 days. I also add a parameter to let us select a date range.

Reply:
We should be using the `Content.PublishedOn` date, not the `FileReference.CreatedOn` date.  The `CreatedOn` date is an audit field that tells us when a row in the database was created.  We  are currently importing historical content, which means all the `CreatedOn` dates will be at the same time.  I suppose for testing purposes this will be okay, but the implementation could be problematic if the performance of S3 isn't ideal.